### PR TITLE
Update readme, small corrections and emojis

### PR DESCRIPTION
**Pull request by sleepypioneer**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,33 +1,40 @@
...
 * [Section 1: Exposing metrics](#section-1:-exposing-metrics)
 * [Section 2: Creating custom metrics](#section-2:-creating-custom-metrics)
 * [Section 3: Scraping Metrics with Prometheus and creating Dashboards with Grafana](#section-3:-scraping-metrics-with-prometheus-and-creating-dashboards-with-grafana)
 * [Bonus Material: Histograms in Prometheus](#bonus-material:-histograms-in-prometheus)
 
-## Prerequisites
+### Prerequisites
 
-For this workshop you will need [Python 3](https://installpython3.com/), [Pipenv](https://pipenv.pypa.io/en/latest/install/#installing-pipenv) and [Docker](https://docs.docker.com/get-docker/) running on your machine.
+For this workshop you will need [Python 3](https://installpython3.com/), [Poetry](https://python-poetry.org/docs/#installation), [Docker](https://docs.docker.com/get-docker/) and [Docker Compose](https://docs.docker.com/compose/install/) running on your machine. *(on mac os docker-compose is by default installed with Docker)*
```
Comment (Vinesse) [inline README.md:21]: Ah okay, good that I see this! I was going to add it as I was surprised it wasn't there yet. Makes sense <3

File: app/main.py [hunk 2]
```diff
@@ -6,9 +6,11 @@
 from util import artificial_503, artificial_latency
 
 
-HOST_NAME = "0.0.0.0"  # This will map to avialable port in docker
+HOST_NAME = '0.0.0.0' # This will map to available port in docker
```
Comment (sleepypioneer) [inline app/main.py]:
```suggestion
HOST_NAME = "0.0.0.0"  # This will map to available port in docker
```

**Comment by sleepypioneer** [comment]
I would go ahead and merge this in @Vinesse unless you have any questions/ suggested changes?

File: README.md [hunk 3]
```diff
@@ -43,36 +50,43 @@ Now try restarting the server (`control c` will stop it) and go to `localhost:80
 
 ---
 
-### Section 2: Creating custom metrics
+### Section 2: Creating custom metrics 🔧
 
 ---
 
-Now that we can expose metrics, we need to create them. Prometheus has a few different data types but the simplest is a `Counter` - this is a counter which always goes up, and can be used to track, for example, the number of requests received (you can then divide this unit over time to calculate requests per second). To create a `Counter`, import it from the Prometheus Python client and instanstiate it.
+Now that we are able to expose metrics we want to also be able to create them. Prometheus has a few different data types, but the most straight forward is a `Counter` - this is a counter which always goes up, and can be used to track, for example, the number of requests received (you can then divide this unit over time to calculate requests per second). To create a `Counter`, import it from the Prometheus Python client and instantiate it.
```
Comment (Vinesse) [inline README.md]:
```suggestion
Now that we can expose metrics, we need to create them. Prometheus has a few different data types, but the most straight forward is a `Counter` - this is a counter which always goes up, and can be used to track, for example, the number of requests received (you can then divide this unit over time to calculate requests per second). To create a `Counter`, import it from the Prometheus Python client and instantiate it.
```

**Pull request merged**